tests/unit/test_report.py

```
@@ -105,18 +111,73 @@ def test_build_no_preloaded_results_does_not_short_circuit_backend(self):
         assert 'info' in report.data
 
 
+class TestReportBuildFromStore:
+	"""Step 4: with the inter-task payload dropped, tasks return topology-only, so the
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)